jooby-caffeine/README.md

```diff
@@ -0,0 +1,128 @@
+# caffeine
```

Comment:
move this file to `md/doc/caffeine/caffeine.md`. The release tool will create this file later.